### PR TITLE
RF | New view > Create new view > Test that some of views were created

### PR DESCRIPTION
**Pull request by EkaterinaTurgunova**
[Trello](https://trello.com/c/53TGRhs6/866-rf-110102-new-view-create-new-view-test-that-some-of-views-were-created)
- Refactor testCreateMyViews() (Class NewView1Test) to POM
- Add new Classes MyViewsPage, NewViewPage, EditViewPage according to POM

### CODE REVIEW

**Comment by EkaterinaTurgunova** [comment]
[RF | New view > Create new view > Test that some of views were created](https://trello.com/c/53TGRhs6/866-rf-new-view-create-new-view-test-that-some-of-views-were-created)

File: src/test/java/NewView1Test.java [hunk 1]
```diff
@@ -75,28 +78,57 @@ public void goToEditView(String viewName) {
...
-        createAnyView(LIST_VIEW_NAME,
-                By.cssSelector("label[for='hudson.model.ListView']"));
-        createAnyView(MY_VIEW_NAME,
-                By.cssSelector("label[for='hudson.model.MyView']"));
-        getDriver().findElement(MY_VIEWS).click();
-
-        Assert.assertTrue(getListViewsNames().contains(GLOBAL_VIEW_NAME));
-        Assert.assertTrue(getListViewsNames().contains(LIST_VIEW_NAME));
-        Assert.assertTrue(getListViewsNames().contains(MY_VIEW_NAME));
+        MyViewsPage myViewsPage = new MyViewsPage(getDriver());
```
Comment (olpolezhaeva) [inline src/test/java/NewView1Test.java:81]: Эта строчка кода не нужна.

File: src/test/java/NewView1Test.java [hunk 2]
```diff
@@ -75,28 +78,57 @@ public void goToEditView(String viewName) {
...
+
+        new HomePage(getDriver())
```
Comment (olpolezhaeva) [inline src/test/java/NewView1Test.java:83]: Здесь в переменную того типа страницы записываем, от которой в ассерте будут идти проверки. То есть MyViewsPage myViewsPage = new HomePage(getDriver())
                                                         .clickNewItem()

File: src/test/java/model/MyViewsPage.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    public HomePage clickDashboard() {
+        topMenuRoot.click();
+
+        return new HomePage(getDriver());
+    }
+
+    public List<WebElement> getListViews() {
+
+        return getDriver().findElements(
+                By.cssSelector(".tabBar .tab a[href]"));
```
Comment (olpolezhaeva) [inline src/test/java/model/MyViewsPage.java:37]: Нужно перенести наверх в виде:
@FindBy(css = ".tabBar .tab a[href]")
private List<WebElement> listViews;

Comment (EkaterinaTurgunova) [reply src/test/java/model/MyViewsPage.java]: Сделала.

**Review by Wis13 — APPROVED** [review]

**Review by InnaRomankova — APPROVED** [review]

**Pull request merged**